### PR TITLE
Spring implementation

### PR DESCRIPTION
**Pull request by BogdanIrimie**
Todo example using spring and spring security with the Aserti Authorization Filter

blocked by: https://github.com/aserto-dev/aserto-console-backend/issues/163
closes https://github.com/aserto-dev/workspace/issues/260

### CODE REVIEW

File: src/main/java/com/aserto/config/Security.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+import org.springframework.security.config.annotation.web.configurers.AbstractHttpConfigurer;
+import org.springframework.security.web.SecurityFilterChain;
+
+@Configuration
+public class Security {
+    @Bean
+    public SecurityFilterChain filterChain(HttpSecurity http) throws Exception {
+        http.cors().and()
+                .csrf(AbstractHttpConfigurer::disable)
+//                .antMatcher("{*path}")
```
[the remark this answers is not in the document]
Comment (BogdanIrimie) [reply src/main/java/com/aserto/config/Security.java]: Was a leftover, removed it

File: src/main/java/com/aserto/config/TodoOwnerResourceMapper.java [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+    private final TodoStore todoStore;
+    private final RequestMappingHandlerMapping handlerMapping;
+
+    public TodoOwnerResourceMapper(TodoStore todoStore, RequestMappingHandlerMapping handlerMapping) {
+        this.todoStore = todoStore;
+        this.handlerMapping = handlerMapping;
+    }
+
+    @Override
+    public Map<String, Value> getResource(HttpServletRequest request) {
```
[the remark this answers is not in the document]
Comment (BogdanIrimie) [reply src/main/java/com/aserto/config/TodoOwnerResourceMapper.java:25]: Added a path parameter resource mapper and made it the default in the [spring-middleware](https://github.com/aserto-dev/aserto-spring/commit/73739d8660f2ab5d93dc0d3f237e43026cf52276)

File: src/main/resources/application.properties.example [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+# --- Directory configuration
+aserto.directory.serviceUrl=localhost:9292
+aserto.directory.insecure=false
+
+# Topaz directory
+aserto.directory.grpc.caCertPath=${user.home}/.config/topaz/certs/grpc-ca.crt
+
+## Aserto hosted directory
+#aserto.directory.apiKey=<api_key>
+#aserto.directory.token=<auth_token>
```
Comment (gimmyxd) [inline src/main/resources/application.properties.example:29]: Why do we have token and api key for directory?

Comment (BogdanIrimie) [reply src/main/resources/application.properties.example:29]: The token should not be there, only the apiKey.

Comment (BogdanIrimie) [reply src/main/resources/application.properties.example]: Removed `aserto.directory.token`

**Review by ronenh — APPROVED** [review]

**Pull request merged**